R/sample_data.R

```diff
@@ -63,9 +63,18 @@ read_sample_data <- function(p, results_dir, tx2gene = NULL) {
       dragen_fusions <- NULL
     }
   }
-  salmon <- salmon_counts(salmon, tx2gene = tx2gene)
   dragen_fusions <- dragen_fusions_read(dragen_fusions)
   dragen_mapping_metrics <- dragen_mapping_metrics_read(dragen_mapping_metrics)
+  #---- Kallisto ----#
+  kallisto <- p[["kallisto"]]
+
+  # check which quant input is provided
+  if (!isnull(salmon)) {
+    counts <- salmon_counts(salmon, tx2gene = tx2gene)
+  } else if ((!isnull(kallisto))) {
```

Comment:
This should be `is.null`

Reply:
https://github.com/umccr/RNAsum/commit/6c6243d2665fe4ed38232421f0a798536a6d6c97